tfmkt/spiders/competitions.py

```diff
@@ -58,7 +58,7 @@ def parse_competitions(self, response, base):
     """Parse competitions from the country competitions page.
 
     @url https://www.transfermarkt.co.uk/wettbewerbe/national/wettbewerbe/157
-    @returns items 3 3
+    @returns items 2 2
```

Comment:
I think you might have an outdated cached response for this check. The number of competitions returned by the function recently changed from 2 to 3, since the Domestic Super Cup for Spain was played last weekend.

You can clear the cache by removing the `.scrapy/httpcache/` folder in the project root.